client/src/components/screens/PatientSubmission/components/ClinicalInformation/index.jsx

```diff
@@ -372,12 +372,12 @@ class ClinicalInformation extends React.Component {
       const { TreeNode } = Tree;
       if (item.children) {
         return (
-          <TreeNode title={item.title} key={item.key} dataRef={item}>
+          <TreeNode title={item.title} key={item.key} dataRef={item} checkable={item.checkable === true}>
```

Comment:
maybe `checkable={item.checkable}`?

Reply:
`checkable` is `undefined` in some cases and I want it to be `false` in this case, not `undefined` (it will default to true if it's the case)